Cargo.lock

```diff
@@ -23,9 +23,9 @@ dependencies = [
 
 [[package]]
 name = "anstream"
-version = "0.5.0"
+version = "0.6.4"
```

Comment:
This feels odd as a separate commit? Presumably some of these changes should be included in the previous commit that bumps the webpki alpha, and this PR otherwise doesn't really need to update unrelated dependencies?

Reply:
ah, i did just plain `cargo update`. have reworked the two cargo.lock commits to make minimal changes, and squashed them into the previous commits (though i'm expecting that this will make rebasing very annoying)